public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
-  
-  We see two built-in structural directives at play in our [example](#template) template:
-+makeExample('architecture/ts/app/hero-list.component.1.html', 'structural')(format=".")
+
+  Our [example](#templates) template uses two built-in structural directives:
+
++makeExcerpt('app/hero-list.component.1.html', 'structural')
+
 :marked
-  * [`*ngFor`](displaying-data.html#ngFor) tells Angular to stamp out one `<div>` per hero in the `heroes` list.
+  * [`*ngFor`](displaying-data.html#ngFor) tells Angular to stamp out one `<li>` per hero in the `heroes` list.
   * [`*ngIf`](displaying-data.html#ngIf) includes the `HeroDetail` component only if a selected hero exists.
-  
-  **Attribute** directives alter the appearance or behavior of an existing element. 
-  In templates they look like regular HTML attributes, hence the name.
-  
-  The `ngModel` directive, which implements two-way data binding, is an example of an attribute directive.
-+makeExample('architecture/ts/app/hero-detail.component.html', 'ngModel')(format=".")
+
+block dart-bool
+  //- N/A
+
 :marked
-  It modifies the behavior of an existing element (typically an `<input>`) 
+  **Attribute** directives alter the appearance or behavior of an existing element.
+  In templates they look like regular HTML attributes, hence the name.
+
+  The `ngModel` directive, which implements two-way data binding, is
+  an example of an attribute directive. `ngModel` modifies the behavior of
+  an existing element (typically an `<input>`)
   by setting its display value property and responding to change events.
-  
-  Angular ships with a few other directives that either alter the layout structure 
-  (e.g. [ngSwitch](template-syntax.html#ngSwitch))
-  or modify aspects of DOM elements and components 
-  (e.g. [ngStyle](template-syntax.html#ngStyle) and [ngClass](template-syntax.html#ngClass)).
-  
-  And of course we can write our own directives.
+
++makeExcerpt('app/hero-detail.component.html', 'ngModel')
+:marked
+  Angular ships with a small number of other directives that either alter the layout structure
+  (for example, [ngSwitch](template-syntax.html#ngSwitch))
+  or modify aspects of DOM elements and components
+  (for example, [ngStyle](template-syntax.html#ngStyle) and [ngClass](template-syntax.html#ngClass)).
+
+  Of course, we can also write our own directives. Components such as
+  `HeroListComponent` are one kind of custom directive.
+  <!-- PENDING: link to where to learn more about other kinds! -->
 
 .l-main-section
-<a id="service"></a>
 :marked
-  ## The Service
+  ## Services
 figure
   img(src="/resources/images/devguide/architecture/service.png" alt="Service" style="float:left; margin-left:-40px;margin-right:10px" )
 :marked
-  "Service" is a broad category encompassing any value, function or feature that our application needs.
-  
-  Almost anything can be a service. 
+  _Services_ is a broad category encompassing any value, function, or feature that our application needs.
```

Comment:
_Service_ is a ...

I don't mind that it is not plural like the section header.